### PR TITLE
Feature: get user info

### PR DESCRIPTION
**Pull request by hoanghong0902**

### CODE REVIEW

**Review by DUYNGOC97 — APPROVED** [review]

File: src/main/java/com/codegym/aurora/converter/UserConverter.java [hunk 1]
```diff
@@ -1,10 +1,11 @@
 package com.codegym.aurora.converter;
 
 import com.codegym.aurora.entity.User;
+import com.codegym.aurora.entity.UserDetail;
 import com.codegym.aurora.payload.response.UserResponseDTO;
 
 public interface UserConverter {
 
-    UserResponseDTO converterEntityUserToUserInfoResponseDTO(User user);
+    UserResponseDTO converterEntityUserToUserInfoResponseDTO(User user, UserDetail userDetail);
```
Comment (ChungNguyenVr912) [inline src/main/java/com/codegym/aurora/converter/UserConverter.java:9]: Invalid arguments, please check...

File: src/main/resources/db/manual/aurorashop.sql [hunk 2]
```diff
@@ -20,7 +20,6 @@ CREATE TABLE USER_DETAIL
     GENDER       NVARCHAR(10),
     PHONE_NUMBER VARCHAR(20),
     EMAIL        VARCHAR(255) UNIQUE,
-    IMAGE_URL    TEXT,
```
Comment (ChungNguyenVr912) [inline src/main/resources/db/manual/aurorashop.sql:23]: Do not modify old sql script!!!

File: src/main/resources/db/migration/V1_00__Create_User_Table.sql [hunk 3]
```diff
@@ -17,7 +17,6 @@ CREATE TABLE USER_DETAIL
     GENDER       NVARCHAR(10),
     PHONE_NUMBER VARCHAR(20),
     EMAIL        VARCHAR(255) UNIQUE,
-    IMAGE_URL    TEXT,
```
Comment (ChungNguyenVr912) [inline src/main/resources/db/migration/V1_00__Create_User_Table.sql:20]: Do not modify old sql script!!!

**Review by ChungNguyenVr912 — CHANGES_REQUESTED** [review]

**Review by mainhuitvt96 — APPROVED** [review]

File: src/main/resources/db/migration/V1_00__Create_User_Table.sql (hunk 3 above)
Comment (hoanghong0902) [reply src/main/resources/db/migration/V1_00__Create_User_Table.sql:20]: fixed

File: src/main/resources/db/manual/aurorashop.sql (hunk 2 above)
Comment (hoanghong0902) [reply src/main/resources/db/manual/aurorashop.sql:23]: fixed

**Pull request merged**